README.md

```diff
@@ -18,6 +18,14 @@ There are two variables that should be set for the Docker-Compose file. The foll
 
 This docker-compose includes the Celery worker that was introduced with Nautobot 1.1.0. Please comment out or remove the celery worker in the `docker-compose.yml` if you are using a pre-1.1.0 release.
 
+## NOTE - MySQL
+
+If you want to use MySQL for the database backend instead of postgres, Please comment out the postgres service and volume and uncomment the mysql service and volume in the `docker-compose.yml`, You will also need to set the following in `local.env`
+```
+NAUTOBOT_DB_ENGINE=django.db.backends.mysql
+NAUTOBOT_DB_HOST=mysql
```

Comment:
maybe using a generic name such as `db` we can omit this change in the variable

Reply:
Might be better to move the engine env to the service definition itself, to prevent the need for changes?